### DISCUSSION TITLE
Success on > 8,000 m peaks in Nepal without bottled oxygen is rare

### DISCUSSION THREAD

**Discussion by mathiselling** [has2k1/plotnine#823] [2024 Plotnine Contest]
### Authors

Mathis Elling

### Links

[linkedIn](https://linkedin.com/in/mathis-elling), [github](https://github.com/mathiselling)

### Full description

You can find the description of my submission on [posit connect cloud](https://connect.posit.cloud/mathiselling/content/01907ab6-8924-f347-f9ab-e359ad93a6ab).

![PNG Submission Mathis](https://github.com/mathiselling/plotnine_contest/blob/main/mathis_plotnine_contest.png?raw=true)

### Code repository

https://github.com/mathiselling/plotnine_contest

**Comment by has2k1** [comment]
![mathis_plotnine_contest](https://github.com/user-attachments/assets/17888f75-2c6f-4f43-aadf-f5d203028ce1)

**Comment by EconomiCurtis** [comment]
Ahoy @mathiselling , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@mathiselling, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
